src/components/NgoDetailsDirection/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import GoogleMapReact from "google-map-react";
+import LocationPin from "../LocationPin";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+const NgoDetailsDirection = ({ address }) => {
+  const { t } = useTranslation();
+  const location = {
```

Comment:
what does this location represent, variable name could be more clear

Reply:
Location of the ngo in the map. I changed the name